microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+![Shopping and coupons](./media/inboxcoupons.png)
+
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and is accessible to you. Inbox coupons will not appear when using **Guest mode** or **InPrivate**.
+
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Click on the **“Settings”** link and view retailers in your inbox 
+1.  Control retailer email preferences via toggles 
+
+To turn off Inbox coupons, completing the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
+1.  Go to the Inbox coupons card and click on the “...” button 
+1.  Select the email account you wish to disconnect and click on the **“Settings”** link 
+1.  **“Disconnect email”**
```

Comment:
```suggestion
To turn off Inbox coupons:

1.  On the **Shopping** sidebar, click the **Your Shopping** tab.

1.  In the **Inbox coupons** card, click the **Settings** (**...**) button.

1.  Select an email account, and then click the **Settings** link.

1.  Click the **Disconnect email** button.
```

Reply:
@mikehoffms the disconnect email is a button (in css) but appears as a link

![image](https://github.com/MicrosoftDocs/edge-developer/assets/133438973/5db97c89-830c-43e9-89b2-7a30ad31b160)

![image](https://github.com/MicrosoftDocs/edge-developer/assets/133438973/08b14497-ebe2-4b41-9c4f-b9a24f4be4e9)